packages/server/src/schema/resolvers.ts

```diff
@@ -26,6 +26,10 @@ export const createSimulation: Resolver<CreateSimulationParameters, SimulationSt
     let id = newid();
     let simulation = atom.slice("simulations", id);
 
+    if(!!simulation.get()) {
+      simulation.remove();
+    }
+
```

Comment:
Why is this necessary?

Reply:
It's from cypress-example branch, I can remove it.